packages/kitten-scientists/source/settings/EngineSettings.ts

```diff
@@ -52,6 +56,7 @@ export class EngineSettings extends Setting {
       this.states.load(settings.states);
       this.language.load(settings.language);
     }
+    this.ksColumn.enabled = localStorage["ks.Column"] === "#ksColumn";
```

Comment:
Don't access localStorage in the loader. Use the .load method of the Setting to copy the state.

The ID of the target column does not need to be stored. You only need the true or false information.